README.md

```diff
@@ -63,10 +63,13 @@ action=create
 u=http://localhost/cgi-bin/csaf_provider.go
 ```
 
-
 ## License
 
 - csaf_distribution is licensed as Free Software under MIT License.
 
 - See the specific source files
-for details, the license itself can be found in the directory `LICENSES`.
\ No newline at end of file
+  for details, the license itself can be found in the directory `LICENSES`.
+
+- csaf_distrubution contains third party Free Software components under compatible licenses, please check [LICENSES/3rdpartylicenses.md](LICENSES/3rdpartylicenses.md) for more info.
```

Comment:
Is the compatibility claim necessary? Given the transitive nature of dependencies and remembering the existence of heated discussions when Facebook initially placed react under some BSD+Patents ("do not sue us") license and upon that community feedback they moved react to MIT I wonder if it would be better, to just leave the assessment of that "compatibility aspect" to users and their legal advisors ...

Reply:
We can improve the phrasing to 'under licenses that to our best knowledge are compatible at time of adding the depenceny` or so.

Reply:
Users usually cannot really assess this, nor can legal advisors (fully). It is a good remark to point out that this compatibility was given to what was visible when we looked it up.

Reply:
That was just an observation from a bystander no need to act on it:
> I do only know some companies, but those I worked in either requested any licenses be a subset of a catalog of "good" licenses or you better implement it yourself. 

Declaring compatibility as vendor of a third party tool (for others) may be as useful as say a risk assessment by a stranger (users do their own assessment or ignore).